Content.Client/Shuttles/UI/ShuttleNavControl.xaml.cs

```diff
@@ -220,7 +220,7 @@ protected override void Draw(DrawingHandleScreen handle)
 
                 var gridCentre = Vector2.Transform(gridBody.LocalCenter, curGridToView);
 
-                var gridDistance = (gridBody.LocalCenter - xform.LocalPosition).Length();
+                var gridDistance = (gridBody.LocalCenter + _transform.GetWorldPosition(grid) - mapPos.Position).Length();
```

Comment:
```suggestion
                var gridDistance = (Vector2.Transform(gridBody.LocalCenter, curGridToWorld) - _transform.GetWorldPosition(xform)).Length();
```

`LocalCenter` is in local space of the grid, so needs to be transformed to the same basis.